lib/csgo_stats/logs.ex

```diff
@@ -19,4 +26,38 @@ defmodule CsgoStats.Logs do
         error
     end
   end
+
+  defp do_parse(entry) do
+    Enum.reduce(entry.lines, [], fn line, acc ->
+      event_text = String.slice(line, 28..-1)
```

Comment:
Let's add a little comment to clarify what we're cutting out.
```suggestion
      # Cut date out of the log. Example: "12/12/2019 - 21:13:56.031 - "
      event_text = String.slice(line, 28..-1)
```